src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -56,7 +56,9 @@
     ComputeModelType,
     DatabaseCapabilitiesAdditionalDetails,
     ElasticPoolCapabilitiesAdditionalDetails,
-    FailoverPolicyType
+    FailoverPolicyType,
+    ManagedInstanceTiers,
```

Comment:
style nitpick: It should be singular, i.g. `ManagedInstanceTier` & `ManagedInstanceFamily`

Reply:
I've removed those because they are not needed any more since I've removed get_enum_type()